demo/visualizer_button.js

```diff
@@ -57,6 +57,16 @@ shakaDemo.VisualizerButton = class extends shaka.ui.Element {
       shakaDemoMain.remakeHash();
       this.setIcon_();
     });
+
+    /** @private {shaka.cast.CastProxy} */
+    this.castProxy_ = this.controls.getCastProxy();
+
+    this.eventManager.listen(this.controls, 'caststatuschanged', () => {
+      if (this.castProxy_.isCasting() &&
```

Comment:
```suggestion
      this.button_.disabled = this.castProxy_.isCasting();
      if (this.castProxy_.isCasting() &&
```

The button should probably be disabled while casting so you can't turn the visualizer back on, right?